applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -371,6 +377,9 @@ void DistanceModificationProcess::RecoverOriginalDistance() {
     mModifiedDistancesValues.resize(0);
     mModifiedDistancesIDs.shrink_to_fit();
     mModifiedDistancesValues.shrink_to_fit();
+
+    // Restore the TO_SPLIT flag original status
+    this->SetContinuousDistanceToSplitFlag();
```

Comment:
this is from a different PR right?

Reply:
Yes, this is from  #4804!

Reply:
#4804 already merged. This should disappear merging the current master.